app/api/webhooks/clerk/route.ts

```
@@ -0,0 +1,201 @@
+
+    if (!svix_id || !svix_timestamp || !svix_signature) {
+      console.error("Missing svix headers");
+      return new Response("Missing svix headers", { status: 400 });
+    }
+
+    // Create webhook instance and verify
+    let event: WebhookEvent;
+    try {
+      console.log(
+        "Attempting to verify webhook with secret starting with:",
+        WEBHOOK_SECRET.substring(0, 5) + "..."
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)